packages/react/src/components/Tag/Tag.js

```diff
@@ -117,6 +146,12 @@ Tag.propTypes = {
    */
   onClose: PropTypes.func,
 
+  /**
+   * Optional prop to render a custom icon.
+   * Can be a React component class
+   */
+  renderIcon: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
```

Comment:
Would there be a reasonable way to just allow teams to render the icon by passing it as a child to the component? Wasn't sure but wanted to check 👀

Reply:
It's possible to do what you described, although it would lack the styles required from the specs, spacing wise.